components/button-toggle-group/src/vwc-button-toggle-group.ts

```diff
@@ -181,16 +171,26 @@ export class VWCButtonToggleGroup extends LitElement {
 			this.clearSelection(buttonElement);
 		}
 		if (this.isButtonValidForToggle(buttonElement)) {
-			toggleButton(buttonElement);
+			this.toggleButtonSelectedState(buttonElement, !isButtonActive(buttonElement));
 			this.dispatchToggleEvent();
 		}
 	}
 
 	private setNodeAttributes(buttonElement: Element) {
-		buttonElement.setAttribute('layout', 'filled');
+		this.toggleButtonSelectedState(buttonElement, isButtonActive(buttonElement));
 		this.toggleChildDisabledState(buttonElement);
 	}
 
+	private toggleButtonSelectedState(buttonElement: Element, isSelected: boolean) {
+		if (isSelected) {
+			buttonElement.setAttribute('layout', 'filled');
```

Comment:
Can we handle this with css instead of mutating properties?

Reply:
not possible only with CSS because the bg property is on the material css. 
I’m keeping the layout change. After the buttons refactor I guess the toggle will be changed as well